frontend/package.json

```diff
@@ -9,7 +9,7 @@
     },
     "scripts": {
         "lint:css": "npx stylelint \"src/**/*.scss\"",
-        "lint:js": "npx eslint \"src/**/*.{ts,tsx}\"",
+        "lint:js": "npx eslint \"src/**/*.{ts,tsx}\" --fix",
```

Comment:
```suggestion
        "lint:js": "npx eslint \"src/**/*.{ts,tsx}\"",
```

Reply:
We do not need to change this command
If you want to run fix locally you can pass params to the command

`npm run lint:js -- --fix`

Even if we need to add such changes, it should be discussed with the team